bundle/regal/ast/imports.rego

```diff
@@ -46,26 +48,31 @@ resolved_imports[identifier] := path if {
 #   returns true if provided path (like ["data", "foo", "bar"]) is in the
 #   list of imports (which is commonly ast.imports)
 imports_has_path(imports, path) if {
-	some imp in imports
+	pv := imports[_].path.value
+
+	count(pv) == count(path)
 
-	_arr(imp) == path
+	every i, part in path {
```

Comment:
Does this run any faster/slower than building an array and using `==`?

Reply:
Potentially, as `every` can benefit from early exit. But it's mostly custom functions that is a win to avoid when they're not strictly needed. 

See https://github.com/open-policy-agent/opa/issues/7266

Reply:
(although this is not a hot path.. I just thought the change made the code nicer :)